miden-tx/src/host/state_tracker.rs

```diff
@@ -51,7 +39,7 @@ impl AccountDeltaTracker {
     }
 
     /// Consumes `self` and returns the resulting [AccountDelta].
-    pub fn into_delta(self) -> AccountDelta {
+    pub fn into_parts(self) -> AccountDelta {
```

Comment:
nit: I would probably keep the name `into_delta()` as it describes the action better.